packages/vite-plugin-svelte/src/index.ts

```diff
@@ -135,7 +143,13 @@ export function svelte(inlineOptions?: Partial<Options>): Plugin[] {
 					}
 				}
 
-				if (ssr && importee === 'svelte') {
+				// TODO: remove this after bumping peerDep on Vite to 4.1+ or Svelte to 4.0+
+				if (
+					viteVersion.startsWith('4.0') &&
+					svelteVersion.startsWith('3') &&
+					ssr &&
+					importee === 'svelte'
```

Comment:
or move the startsWith results to consts?

Reply:
I moved the `startWith` to `const`s. I think it will be more performant to do the Vite comparison first because then a single comparison will usually be enough. If we do the `ssr` comparison first it will be true at most half a time requiring additional comparisons.